exonum-java-binding/core/rust/exonum-java/src/node.rs

```diff
@@ -97,10 +112,9 @@ fn create_database(config: &Config) -> Result<Arc<dyn Database>, failure::Error>
     Ok(database)
 }
 
-fn standard_exonum_service_factories() -> Vec<InstanceCollection> {
+fn standard_exonum_service_factories() -> Vec<Box<dyn ServiceFactory>> {
     // TODO(ECR-3714): add anchoring service
-    vec![
-        InstanceCollection::new(TimeServiceFactory::with_provider(SystemTimeProvider)),
-        InstanceCollection::from(SimpleSupervisor::new()),
-    ]
+    vec![Box::new(TimeServiceFactory::with_provider(
```

Comment:
Every `ServiceFactory` can be boxed with existing `From` implementation but I'm unsure what is more clear - implicit `.into()` or explicit `Box::new()`.

Reply:
Box::new is better here imo